src/routes/safe/components/Transactions/TxList/index.tsx

```diff
@@ -7,14 +7,18 @@ import { SAFE_NAVIGATION_EVENT, useAnalytics } from 'src/utils/googleAnalytics'
 import { HistoryTransactions } from './HistoryTransactions'
 import { QueueTransactions } from './QueueTransactions'
 import { ContentWrapper, Wrapper } from './styled'
+import { createBrowserHistory } from 'history'
+import { TRANSACTIONS_PAGE_TABS } from './utils'
 
 const items: Item[] = [
-  { id: 'queue', label: 'Queue' },
-  { id: 'history', label: 'History' },
+  { id: TRANSACTIONS_PAGE_TABS.QUEUE, label: 'Queue' },
+  { id: TRANSACTIONS_PAGE_TABS.HISTORY, label: 'History' },
 ]
 
 const GatewayTransactions = (): ReactElement => {
-  const [tab, setTab] = useState(items[0].id)
+  const history = createBrowserHistory()
```

Comment:
You shouldn't create a new `history` object. If you are inside a component, you can access it via the `useHistory()` hook, or `import { history } from 'src/routes/routes'` it directly if you are in a function.

Reply:
I am pulling the history object from the _useHistory()_ now 👍